[33mWARNING [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:149 Exception managing chrome: error sending request for url (https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json): error trying to connect: dns error: failed to lookup address information: nodename nor servname provided, or not known
[33mWARNING [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:149 The chromedriver version (120.0.6099.71) detected in PATH at /opt/homebrew/bin/chromedriver might not be compatible with the detected chrome version (121.0.6167.139); currently, chromedriver  is recommended for chrome 121.*, so it is advised to delete the driver in PATH and retry
[33mWARNING [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:149 Error sending stats to Plausible: error sending request for url (https://plausible.io/api/event): operation timed out